ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -837,6 +837,24 @@ def test_tensorflow_adjoint(
     )
 
 
+# qr
+@handle_frontend_test(
+    fn_tree="tensorflow.linalg.qr",
+    dtype_and_input=_get_dtype_and_matrix(),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_qr(
+        *,
+        dtype_and_input,
+        frontend,
+        test_flags,
+        fn_tree,
+        on_device,
+):
+    input_dtype, x = dtype_and_input
+    ret, frontend_ret = helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+=======
```

Comment:
Hi @saravanadivakaruni 

I think you've accidentally added these lines while resolving the merge conflicts. The CI tests are also failing because of this. Could you kindly remove these lines and re-request a review once that's done. Thanks! 🙂

Reply:
I removed those lines, Please let me know if I have to do anything else.

Reply:
They are still showing up in the modified files though

Reply:
I have changed them and commit again

Reply:
its *still* showing up in the modified files 😅